handbook/bizops/index.md

```diff
@@ -29,12 +29,20 @@ Here are the following sources we collect data from:
 * Sourcegraph.com Site-admin pages: customer subscriptions and license keys
 * [Pings](https://docs.sourcegraph.com/admin/pings) from self-hosted Sourcegraph instances containing anonymous and aggregated information
 * [Custom tool to track events](https://github.com/sourcegraph/sourcegraph/issues/5486) on the Sourcegraph.com instance
+* [Prometheus search data](https://sourcegraph.com/-/debug/grafana/d/0ZBSEmRWk/searcher?orgId=1&refresh=5s) represents the breakdown of search statuses (success, error, timeout, partial timeout) on [Sourcegraph.com](https://sourcegraph.com/search)
 
 ### Data tools
 
 * [Looker](#using-looker): Business intelligence/data visualization tool
 * Google Cloud Platform: BigQuery is our data warehouse and the database Looker runs on top of
 * Google Sheets: There are a [number of spreadsheets](https://drive.google.com/drive/folders/1vOyhFO90FjHe-bwnHOZeljHLuhXL2BAv)that Looker queries (by way of BigQuery).
+* BizOps does built ah-hoc tools to analyze data for various reasons. Currently, the only major tool [analyzes Sourcegraph.com search data](https://docs.google.com/spreadsheets/d/1C7ZMcurYv48CmKr_B1_rFNBdFazsxNq5c-wZMlUQjXQ/edit#gid=1679017494). All scripts are in the [analytics repo](https://github.com/sourcegraph/analytics). 
```

Comment:
```suggestion
* BizOps does build ad-hoc tools to analyze data for various reasons. Currently, the only major tool [analyzes Sourcegraph.com search data](https://docs.google.com/spreadsheets/d/1C7ZMcurYv48CmKr_B1_rFNBdFazsxNq5c-wZMlUQjXQ/edit#gid=1679017494). All scripts are in the [analytics repo](https://github.com/sourcegraph/analytics). 
```

Reply:
This is an interesting section... I'm torn between trying to list all scratch analyses we ever make like this one as sub-bullets here, or going the opposite direction and instead not trying to be comprehensive here at all (and instead saying "just check the analytics repo"). What's your take?

Reply:
Actually, just thought of a better solution (I think). I'll link it to a Google Drive folder where we have outputs from tools and also link to the repo for the source code. Thoughts?